protogen/src/main/java/com/netflix/conductor/protogen/ProtoGen.java

```diff
@@ -0,0 +1,104 @@
+package com.netflix.conductor.protogen;
+
+import com.github.jknack.handlebars.EscapingStrategy;
+import com.github.jknack.handlebars.Handlebars;
+import com.github.jknack.handlebars.Template;
+import com.github.jknack.handlebars.io.FileTemplateLoader;
+import com.github.jknack.handlebars.io.TemplateLoader;
+import com.squareup.javapoet.*;
+
+import javax.annotation.Generated;
+import javax.lang.model.element.Modifier;
+import java.io.FileWriter;
+import java.io.Writer;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+
+public class ProtoGen {
+    public static String PROTO_PACKAGE_NAME = "conductor.proto";
+    public static String PROTO_JAVA_PACKAGE_NAME = "com.netflix.conductor.proto";
+    public static String PROTO_GO_PACKAGE_NAME = "github.com/netflix/conductor/client/gogrpc/conductor/model";
+
+    public static String GENERATED_MAPPER_PACKAGE = "com.netflix.conductor.grpc";
+    public static String GENERATOR_NAME = "com.netflix.conductor.protogen.ProtoGen";
+
+    private List<File> files = new ArrayList<>();
+
+    public static void main(String[] args) throws Exception {
```

Comment:
Is this integrated with build process, or executed manually?

Reply:
It's run manually right now. I'll try to wire this up to run automatically as soon as I extract the ProtoGen code into its own repository.